### ISSUE TITLE
Closure Compiler Native Image Builds on Windows

### ISSUE THREAD

**Issue by ChadKillingsworth** [oracle/graal#1137]
As soon as it's appropriate, I'd like to help test the native image builds on Windows for Closure Compiler. I don't mind to build Graal from source to do that.

https://twitter.com/thomaswue/status/1114449543729430528?s=09

**Comment by thomaswue** [comment]
@pejovica Can you check if the current Windows support for native image is sufficient for building @ChadKillingsworth 's program?

**Comment by pejovica** [comment]
@ChadKillingsworth I've tried the [Hello World example](https://developers.google.com/closure/compiler/docs/gettingstarted_app), and it seems to work well. It won't work out-of-the-box because it needs additional configuration, but that's not Windows specific.

Getting the config for resources was straightforward, however, I used the [native-image-agent](https://github.com/oracle/graal/blob/master/substratevm/CONFIGURE.md) to collect the reflection config as `args4j` seems to rely heavily on reflection.

With the config in place, you can use the following command to build an image:
```
mx native-image -jar compiler.jar -H:ConfigurationFileDirectories=config
```

And here are the [config files](https://github.com/oracle/graal/files/3054550/config.zip) I used for both Windows and Linux.

**Comment by ChadKillingsworth** [comment]
Thanks I will give it a go. I already have the reflection config from generating images for Unix and macos.

**Comment by ChadKillingsworth** [comment]
What jdk is currently recommended for building substratevm?

**Comment by pejovica** [comment]
You need a JVMCI-enabled JDK 8 that has the necessary native libraries available for static linking. I used `labsjdk`, which can be downloaded from [OTN](https://www.oracle.com/technetwork/oracle-labs/program-languages/downloads/index.html), but OpenJDK based [build](https://github.com/graalvm/openjdk8-jvmci-builder/releases) should also work.

**Comment by ChadKillingsworth** [comment]
We've successfully built an published a windows version using the 1.0 release.
